### PR TITLE
[IMP] Helpdesk: update basics 18

### PR DESCRIPTION
**Pull request by jero-odoo**
Updating Helpdesk basics doc for V18. Adding section on new feature - "Converting tickets to CRM opps"

### CODE REVIEW

**Comment by jero-odoo** [comment]
Hi @benderliz this is ready for peer review. Thanks!

File: content/applications/services/helpdesk.rst [hunk 1]
```diff
@@ -5,20 +5,19 @@
 Helpdesk
 ========
 
-Odoo *Helpdesk* is a ticketing-based customer support application. Multiple teams can be configured
-and managed in one dashboard, each with their own pipeline for tickets submitted by customers.
-Pipelines are organized in customizable stages that enable teams to track, prioritize, and solve
-customer issues quickly and efficiently.
+Odoo **Helpdesk** is a ticketing-based customer support application. Multiple teams can be
```
Comment (benderliz) [inline content/applications/services/helpdesk.rst:8]:
```suggestion
Odoo **Helpdesk** is a ticket-based customer support application. Multiple teams can be
```

File: content/applications/services/helpdesk.rst [hunk 2]
```diff
@@ -5,20 +5,19 @@
...
+configured and managed in one dashboard, each with their own pipeline for tickets submitted by
+customers. Pipelines are organized in customizable stages that enable teams to track, prioritize,
+and solve customer issues quickly and efficiently.
 
 Create a Helpdesk team
 ======================
 
-To view or modify *Helpdesk* teams, go to :menuselection:`Helpdesk app --> Configuration -->
+To view or modify **Helpdesk** teams, go to :menuselection:`Helpdesk app --> Configuration -->
 Helpdesk Teams`. To create a new team, click the :guilabel:`New` button in the top-left of the
```
Comment (benderliz) [inline content/applications/services/helpdesk.rst:17]:
```suggestion
Helpdesk Teams`. To create a new team, click the :guilabel:`New` button on the top-left of the
```

File: content/applications/services/helpdesk.rst [hunk 3]
```diff
@@ -5,20 +5,19 @@
...
 dashboard.
 
 .. image:: helpdesk/helpdesk-teams-list.png
-   :align: center
    :alt: View of the Helpdesk teams page in Odoo Helpdesk.
 
 On the blank Helpdesk team form, enter a :guilabel:`Name` for the new team. Then, enter a
```
Comment (benderliz) [inline content/applications/services/helpdesk.rst:23]: It's not letting me suggest directly, but in line 24, it says, "...in the field below the team name," - does this field have a name that can be used with :guilabel:?

Also, in the 'important' admonition block, I would probably revise lines 28-29 for conciseness:

The team description is published on the public-facing :doc:`ticket submission form
   <helpdesk/overview/receiving_tickets>`.

File: content/applications/services/helpdesk.rst [hunk 4]
```diff
@@ -32,7 +31,6 @@ team is assigned to, select it from the :guilabel:`Company` drop-down menu.
    use only.
 
    .. image:: helpdesk/team-description-webform.png
-      :align: center
       :alt: View of a Helpdesk team's website form displaying the team description.
 
 Visibility & Assignment
```
Comment (benderliz) [inline content/applications/services/helpdesk.rst:36]: I know we tend to de-capitalize text the after colons in bullet points, but because they're sentences, I would capitalize all of them. For example, in line 48, I would make it: 

:guilabel:`Invited internal users (private)`: Internal users can access the team and the tickets

Comment (benderliz) [inline content/applications/services/helpdesk.rst:36]: For the warning block starting in line 64, I would revise to be more explicit about field changes:

.. warning::
   A team's visibility can be altered after the initial configuration. However, if the team changes
   from public ('Invited portal users and all internal users') access to either private ('Invited internal users') or company- only ('All internal users') access, portal users are removed as followers from both the team, and from individual tickets.

Comment (benderliz) [inline content/applications/services/helpdesk.rst:36]: Line 72: I would re-title this to: Follow a team's tickets

I know Odoo's field is called "Follow All Team's Tickets", but as a header, it sounds a little awkward.

Comment (benderliz) [inline content/applications/services/helpdesk.rst:36]: On line 87, I would reword slightly to: 

manually on each individual ticket, or through :guilabel:`Automatic Assignment`. Check the

File: content/applications/services/helpdesk.rst [hunk 5]
```diff
@@ -90,7 +88,6 @@ manually on each ticket individually, or through :guilabel:`Automatic Assignment
 :guilabel:`Automatic Assignment` checkbox to enable this feature for the team.
 
 .. image:: helpdesk/helpdesk-visibility-assignment.png
-   :align: center
    :alt: View of a Helpdesk team settings page emphasizing the automatic assignment features in Odoo
          Helpdesk.
```
Comment (benderliz) [inline content/applications/services/helpdesk.rst:93]: For line 94, when it says, "additional fields appear", is this just referencing the following assignment method options? If so, then I would just delete this and go straight into the assignment methods. Or be more specific.

Comment (benderliz) [inline content/applications/services/helpdesk.rst:93]: For lines 99 & 102, I would capitalize the words after the colon, since they're full sentences.

File: content/applications/services/helpdesk.rst [hunk 6]
```diff
@@ -143,6 +140,43 @@ drop-down menu. Doing so opens a new page where the selected tickets are listed
...
+At the top of the ticket, click the :guilabel:`Convert to Opportunity` button.
+
+.. note::
+   If :doc:`leads <../sales/crm/acquire_leads/convert>` are enabled on the **CRM** app, tickets are
+   converted to *leads*, and the button reads :guilabel:`Convert to Lead`.
+
+This opens the :guilabel:`Convert to Opportunity` pop-up. Fill in or select the following
+information on the pop-up:
+
+- Customer: select whether to :guilabel:`Create a new customer`, :guilabel:`Link to an existing
```
Comment (benderliz) [inline content/applications/services/helpdesk.rst:164]:
```suggestion
- Customer: Select whether to :guilabel:`Create a new customer`, :guilabel:`Link to an existing
```

File: content/applications/services/helpdesk.rst [hunk 7]
```diff
@@ -143,6 +140,43 @@ drop-down menu. Doing so opens a new page where the selected tickets are listed
...
+  customer`, or :guilabel:`Do not link to a customer`. If :guilabel:`Link to a customer` is chosen,
+  select the appropriate customer name from the :guilabel:`Customer` drop-down.
+
+- Sales Team: select a :guilabel:`Sales Team` and :guilabel:`Salesperson` to assign the opportunity
```
Comment (benderliz) [inline content/applications/services/helpdesk.rst:168]:
```suggestion
- Sales Team: Select a :guilabel:`Sales Team` and :guilabel:`Salesperson` to assign the opportunity
```

File: content/applications/services/helpdesk.rst [hunk 8]
```diff
@@ -143,6 +140,43 @@ drop-down menu. Doing so opens a new page where the selected tickets are listed
...
+  to.
+
+.. image:: helpdesk/convert-to-opp.png
+   :alt: The convert to opportunity pop-up window.
+
+After completing the form, click :guilabel:`Convert to Opportunity`. Doing so creates a new
+opportunity. The original ticket is linked in the chatter of the new opportunity for traceability.
```
Comment (benderliz) [inline content/applications/services/helpdesk.rst:175]:
```suggestion
opportunity in the **Sales** app. The original ticket is linked in the chatter of the new opportunity for traceability.
```

File: content/applications/services/helpdesk.rst [hunk 9]
```diff
@@ -143,6 +140,43 @@ drop-down menu. Doing so opens a new page where the selected tickets are listed
...
+
+.. note::
+   After the ticket is converted to an opportunity, it is archived.
```
Comment (benderliz) [inline content/applications/services/helpdesk.rst:178]:
```suggestion
   After the ticket is converted to an opportunity, the ticket is archived.
```

Comment (benderliz) [reply content/applications/services/helpdesk.rst:178]: Just clarifying that the opportunity is not what gets archived.

**Review by benderliz — APPROVED** [review]
Hi @jero-odoo! You did a fantastic job revising this Helpdesk basics doc. Excellent work adding in the 'Convert tickets to opportunities' section - I know this will help a lot of our users! Your writing is super readable, and great job on all the RST conventions. I made some minor suggestions, which are mostly stylistic or to clarify minor things for the reader. After looking at those, this doc is ready for a quick final review!

File: content/applications/services/helpdesk.rst (hunk 8 above)
Comment (jero-odoo) [reply content/applications/services/helpdesk.rst:175]: Opps and leads are in the CRM app, but I will add this :+1:

File: content/applications/services/helpdesk.rst (hunk 3 above)
Comment (jero-odoo) [reply content/applications/services/helpdesk.rst:23]: No, the field does not have a guilabel

File: content/applications/services/helpdesk.rst [hunk 10]
```diff
@@ -143,6 +139,44 @@ drop-down menu. Doing so opens a new page where the selected tickets are listed
 :guilabel:`Similarity` rating. From here, click either :ref:`Merge <data_cleaning/merge-records>` to
 combine the tickets, or :guilabel:`DISCARD`.
 
+Convert tickets to opportunities
+================================
```
Comment (Felicious) [inline content/applications/services/helpdesk.rst:143]: This section is really cool and such a good add to acknowledge the integrated nature of Odoo. My only feedback is sharing CRM's main doc page or eLearning lesson somewhere in this section

File: content/applications/services/helpdesk.rst [hunk 11]
```diff
@@ -143,6 +139,44 @@ drop-down menu. Doing so opens a new page where the selected tickets are listed
...
+At the top of the ticket, click the :guilabel:`Convert to Opportunity` button.
+
+.. note::
+   If :doc:`leads <../sales/crm/acquire_leads/convert>` are enabled on the **CRM** app, tickets are
+   converted to *leads*, and the button reads :guilabel:`Convert to Lead`.
+
+This opens the :guilabel:`Convert to Opportunity` pop-up. Fill in or select the following
+information on the pop-up:
+
+- Customer: Select whether to :guilabel:`Create a new customer`, :guilabel:`Link to an existing
```
Comment (Felicious) [inline content/applications/services/helpdesk.rst:163]: Since we're directly in the pop-up and referring directly to the field, guilabel seems more suitable (:
```suggestion
- :guilabel:`Customer`: Select whether to :guilabel:`Create a new customer`, :guilabel:`Link to an existing
```

File: content/applications/services/helpdesk.rst [hunk 12]
```diff
@@ -143,6 +139,44 @@ drop-down menu. Doing so opens a new page where the selected tickets are listed
...
+  customer`, or :guilabel:`Do not link to a customer`. If :guilabel:`Link to a customer` is chosen,
+  select the appropriate customer name from the :guilabel:`Customer` drop-down.
+
+- Sales Team: Select a :guilabel:`Sales Team` and :guilabel:`Salesperson` to assign the opportunity
```
Comment (Felicious) [inline content/applications/services/helpdesk.rst:167]: While your original wording is correct, we can make it a bit clearer by focusing on what Odoo does here, rather than just describing the fields. 😊 How about:

```suggestion
- :guilabel:`Sales Team`: Specify which :guilabel:`Sales Team` and :guilabel:`Salesperson` this created opportunity is assigned
```

**Review by Felicious — APPROVED** [review]
Amazing job with this PR, @jero-odoo !

This is a clear doc, and everything worked as expected. I love that you're referencing other relevant apps :smile:

**Comment by jero-odoo** [comment]
thank you!
@samueljlieber this is ready for tech review. Thanks!

**Review by samueljlieber — APPROVED** [review]
Hi @jero-odoo! Great job with this pr, your improvements look good to me! Also nice job on the new section 👍 Thank you for your work!
...
@robodoo r+

**Comment by samueljlieber** [comment]
@robodoo retry

**Comment by StraubCreative** [comment]
@robodoo retry
